test/unit/views/movement/MovementConfirmationDepartureViewSpec.scala

```diff
@@ -16,44 +16,41 @@
 
 package views.movement
 
+import base.Injector
 import forms.{ConsignmentReferenceType, ConsignmentReferences}
 import models.cache.{DepartureAnswers, JourneyType}
 import views.ViewSpec
 import views.html.movement_confirmation_page
 
-class MovementConfirmationDepartureViewSpec extends ViewSpec {
+class MovementConfirmationDepartureViewSpec extends ViewSpec with Injector {
 
   private implicit val request = journeyRequest(DepartureAnswers())
 
   private val consignmentReferences = ConsignmentReferences(ConsignmentReferenceType.D, "9GB12345678")
-  private val page = new movement_confirmation_page(main_template)
+  private val page = instanceOf[movement_confirmation_page]
 
   "View" should {
     "render title" in {
-      page(JourneyType.DEPART, consignmentReferences).getTitle must containMessage("movement.confirmation.DEPART.tab.heading")
+      page(JourneyType.RETROSPECTIVE_ARRIVE, consignmentReferences).getTitle must containMessage("movement.confirmation.title.RETROSPECTIVE_ARRIVE")
```

Comment:
Similar here: you have `JourneyType.RETROSPECTIVE_ARRIVE` but the test suite is for Departures.

Reply:
Changed